perf/stability/redis/README.md

```diff
@@ -0,0 +1,7 @@
+# Redis
+
+This test installs an instance of Redis using the [stable/redis](https://github.com/helm/charts/tree/master/stable/redis) Helm chart.
+
```

Comment:
John, can you update the readme to describe the actual test apart from the setup ?
It is doing a `get_time` call right?

Reply:
I will update the readme. It is just doing a set and get with the time as the value. This is to ensure we aren't failing to write and reading stale entries or something. This has been pretty stable but if it gets flakey we can change it (we want to test Istio not the time it takes Redis to propagate values)